functions-and-operators/string-functions.md

```diff
@@ -71,40 +55,38 @@ The `BIN()` function is used to convert the given argument into a string represe
 - If the argument is a string that consists of digits and non-digits, the function returns the result according to the consecutive digits at the beginning of the argument. For example, the results for `"123q123"` and `123` are the same.
```

Comment:
BIN('123q123') will generate warnings like, better mention it: 
mysql> show warnings;
+---------+------+----------------------------------------------+
| Level   | Code | Message                                      |
+---------+------+----------------------------------------------+
| Warning | 1292 | Truncated incorrect INTEGER value: '123q123' |
+---------+------+----------------------------------------------+
1 row in set (0.00 sec)